OpenSim/Simulation/SimbodyEngine/Test/testJoints.cpp

```diff
@@ -2226,6 +2231,42 @@ void testAutomaticJointReversal()
     ASSERT_EQUAL(accErr, 0.0, sqrt(integ_accuracy));
 }
 
+void testUserJointReversal()
+{
+    using namespace OpenSim;
+
+    cout << "\n==========================================================="
+         << "\n Test joint reversal set by user in old model file"
+         << "\n==========================================================="
+         << endl;
+
+    // Open model.
+    auto model = Model("double_pendulum_testReverse.osim");
+    //model.finalizeFromProperties();
+    model.initSystem();
```

Comment:
I thought `model.finalizeFromProperties()` should work here, but I get this error:
<pre>
Loaded model double_pendulum_testReverse from file double_pendulum_testReverse.osim
PinJoint::Socket 'parent_frame' is not connected to 'ground' of type PhysicalFrame
        In file C:\Users\tkuchida\GitHub\opensim-core\OpenSim/Common/ComponentSocket.h:329
        In function 'getConnectee'
</pre>